style.css

```diff
@@ -369,17 +407,157 @@ hr {
   padding: 12px 16px;
 }
 
+.nav-btn.active li a {
+  font-size: 32px;
+  color: white;
+  font-weight: 600;
+  line-height: 44px;
+  margin-bottom: 20px;
+}
+
+#preview {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(193, 199, 208, 0.9);
+  backdrop-filter: blur(5px);
+  z-index: 7;
+  padding: 10px;
+  margin: 0 auto;
+}
+
+.preview-project {
+  position: fixed;
+  top: 47%;
+  left: 47%;
+  transform: translate(-50%, -50%);
+  box-sizing: border-box;
+  background-color: white;
+  width: 70%;
+  border-radius: 10px;
+  margin: 10px auto;
+  padding: 1rem;
+}
+
+.pre-project-info {
+  display: grid;
+  justify-content: space-between;
+  grid-template-columns: 60% 40%;
+  margin: 0 auto;
+}
+
+.pre-project-info hr {
+  border-top: 2px solid #ebecf0;
+  margin: 5px 0 20px 0;
+}
+
+.preview-project-title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 5px 0;
+}
+
+.preview-title-div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.preview-project-xmark {
+  position: relative;
+  bottom: 15px;
+  font-size: 14px;
+  border: none;
+  background-color: white;
+  color: #67798e;
+  cursor: pointer;
+}
+
+.preview-project-img {
+  width: 100%;
+  height: 53vh;
+}
+
+.preview-btn {
+  font-family: "Poppins", sans-serif;
+  text-decoration: none;
+  font-weight: 400;
+  line-height: 20px;
+  font-size: 14px;
+  padding: 10px 15px;
+  border-radius: 10px;
+  background-color: white;
+  border: 1px solid var(--primary);
+  color: var(--primary);
+  cursor: pointer;
+  margin-bottom: 10px;
+  margin-right: 5px;
+}
```

Comment:
dear code reviewer first of all I could not find the icon of the pop up window button icons in fontawesme so when the button is hover  the icon color can be change to white as like text so i decide to take that icon from Figma but when I try with that when the button is hover or pressed the icon was disappearing because the icon is in png or svg format.

Reply:
I am totally disagree with you for the changes you requested me because all the change you said it was not necessary and I tried all of them.

Reply:
Dear code reviewer firstly I try to find the buttons icon in fontawesome but there was not you can check it then I decided to take the icon form the Figma in png or svg formats when I set the Figma Icon and give the button hover and pressed classes the the icons was disappearing because the icon is in the png format it is not like the text so when it is hovered of pressed it can change it's color.

Reply:
I totally disagree with your coding review I tried all the required change you mentioned. any way I am do as you say but I believe these required changes you mentioned was not necessary at all